great_expectations/validator/validator.py

```diff
@@ -766,6 +766,7 @@ def resolve_validation_graph(
         graph: ValidationGraph,
         metrics: Dict[Tuple[str, str, str], Any],
         runtime_configuration: Optional[dict] = None,
+        min_graph_edges_pbar_enable: int = 0,  # Set to low number (e.g., 3) to suppress progress bar for small graphs.
```

Comment:
Nitpick - could we possibly rename this? Maybe something like `progress_bar_edge_threshold`? I'm not the best with naming 😄

Reply:
Why do we count the edges and not the nodes of the graph anyway?

Reply:
@cdkini I do not mind a better name; for now -- since the logic talks about edges, let us leave like this -- we can change later once a potentially better name is confirmed.  Thanks!